src/scss/settings/_variables.scss

```diff
@@ -117,9 +117,9 @@ $zIndex: (
 
 // Path vars
 // ==========================================================================
-$img-path                  : '/assets/dist/img'; // Usage: background: url('#{$img-path}/img.jpg') 0 0 no-repeat;
+$img-path                  : './dist/img'; // Usage: background: url('#{$img-path}/img.jpg') 0 0 no-repeat;
```

Comment:
These paths should definitely have `!default` after them so they can be overridden potentially within a project.

Ideally, the assets should be copied down from Fozzie to the base project asset folder, otherwise, the image paths will end up being inconsistent between the project and each dependency that references its own assets.

Think we chatted about [this sort of task](https://www.npmjs.com/package/npm-assets) being built into the gulp tasks to help with this.